templates/plugin/ping.conf.erb

```diff
@@ -1,27 +1,23 @@
-# Generated by Puppet
-LoadPlugin "ping"
```

Comment:
Yes. It's automatically added on to it by the `collected::plugin` defined type. It includes that part of the template: https://github.com/puppet-community/puppet-collectd/blob/master/manifests/plugin.pp